src/components/Routes/Routes.js

```diff
@@ -25,9 +26,10 @@ const Routes = () => (
         <Switch location={location}>
           <DefaultLayout path="/" exact component={HomePage} />
           <DefaultLayout path="/home" component={HomePage} />
-          <DefaultLayout path="/product-details" component={ProductDetailsPageWrapper} />
           <DefaultLayout path="/WhatIsNew" component={WhatIsNew} />
           <DefaultLayout path="/MostPopular" component={MostPopular} />
+          <DefaultLayout path="/cart" component={CartPage} />
+          <DefaultLayout path="/:id" exact component={ProductDetailsPageWrapper} />
```

Comment:
why not:
```jsx
<DefaultLayout path="products/:id" exact component={ProductDetailsPageWrapper} />
```

Reply:
It just didn't work that way, no idea why
trying to figure it out

Reply:
I don't really need it, i will remove it